data/json/items/comestibles.json

```diff
@@ -1129,8 +1129,8 @@
         "volume" : 1,
         "phase" : "liquid",
         "charges" : 7,
-        "flags": [ "EATEN_COLD" ],
-        "fun" : 12
+        "fun" : 12,
+        "flags": [ "EATEN_COLD" ]
```

Comment:
>all wine is served cold

That's a big negative right there, pilgrim; wine is served at a variety of temperatures. Sometimes it's even served hot.